compiler/z/codegen/UnaryEvaluator.cpp

```diff
@@ -77,7 +77,7 @@ extern void PRINT_ME(char * string, TR::Node * node, TR::CodeGenerator * cg);
 TR::Register *
 lconstHelper(TR::Node * node, TR::CodeGenerator * cg)
    {
-   TR_ASSERT( TR::Compiler->target.is32Bit() || cg->evaluateNodeInRegPair(node), "lconstHelper() is for 32bit only!");
+   TR_ASSERT( TR::Compiler->target.is32Bit(), "lconstHelper() is for 32bit only!");
```

Comment:
May as well properly format this while we're in the area (extra space after `TR_ASSERT`).